### ISSUE TITLE
Mismatch between definition and access of `width`

### ISSUE THREAD

**Issue by leahneukirchen** [jgdavey/prose#5]
prose 0.4.0 compiled with rustc 1.68.2 and clap 4.0.29 yields:

```
% prose
thread 'main' panicked at 'Mismatch between definition and access of `width`. Could not downcast to TypeId { t: 9919015677752515272 }, need to downcast to TypeId { t: 7761400204611230935 }
', /host/cargo/registry/src/index.crates.io-6f17d22bba15001f/clap-4.0.29/src/parser/error.rs:30:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by leahneukirchen** [comment]
Nevermind, this is fixed in 037bf5b8e14fc0b8b25e00e2e076515148860825. Please tag a new release maybe?

**Comment by jgdavey** [comment]
Thanks for pointing this out. I've just released version 0.4.2 which should handle the issue!
